README.md

```diff
@@ -21,3 +20,45 @@ Have a look at the file for details, but in short this is how it works:
 - Create a custom docker image for Drupal and nginx, and push those to a docker registry (typically that of your cloud provider).
 - Install or update our helm chart while passing our custom images as parameters.
 - The helm chart executes the usual drush deployment commands.
+
+## Local development
+
+Install [Lando](https://docs.devwithlando.io/). Add your project's name and other project-specific settings to `.lando.yml` and run `lando start`.
+
+Useful helper commands:
+
+- `lando cre` runs `composer require`
```

Comment:
I'm personally not a fan of wrapping common commands with an alias just to save keystrokes. It's confusing for newcomers, and it hides what's happening.

Reply:
Ok, I'll remove those ;)